packages/odf/components/create-storage-system/external-systems/common/select-local-cluster-nodes-table/select-local-cluster-nodes-table.scss

```
@@ -0,0 +1,11 @@
+.odf-capacity-and-nodes__select-nodes {
+  .odf-m-pane__body {
+    /* Removes padding from the list page to align the table with rest of the form fields*/
```

Stylelint (17.14.1)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)